scripts/MappingCodeOct.R

```diff
@@ -0,0 +1,199 @@
+library('tidyverse')
+library('sf')
+library('scales')
+
+## ONS provides ICS geographical data as a GEOsjon on Open Geography portal (https://geoportal.statistics.gov.uk/)
+ICS <- st_read('https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Integrated_Care_Boards_July_2022_EN_BFC_2022/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson')
+
+#check geographical data works
+# plot(st_geometry(ICS))
+
+## import correct Oxybate data mapped to ICB '22 names
+# ICS data from ONS uses 2021 onward ICS names which don't match with SCMD ICS/STP names
+# Manually corrected in file below
+
+MapData <- read.csv("C:/../MappingOxybate.csv")
```

Comment:
use here package for relative file paths https://here.r-lib.org/

Reply:
for example see here https://github.com/ebmdatalab/scmd-narcolepsy/blob/29101a46fd3b0f55dfae23707e94310440d86cfa/notebooks/scmd-narcolepsy-meds.Rmd#L70-L72